pyfstat/make_sfts.py

```diff
@@ -775,29 +775,25 @@ def parse_args_consistent_with_mfd(self):
             "period",
             "transientWindowType",
         ]
-
-        signal_parameters = {
-            key: self.__dict__.get(key, None) for key in signal_parameter_labels
-        }
-        self.signal_parameters = {
-            key: value for key, value in signal_parameters.items() if value is not None
+        gps_time_and_string_formats = {
+            # GPS times should NOT be parsed using scientific notation
+            # LAL routines silently parse them wrongly
+            "orbitTp": ":10.6f",
+            "refTime": ":10.6f",
+            "transientWindowType": ":s",
         }
 
-        signal_parameter_formats = (len(signal_parameter_labels) - 1) * [":1.18e"] + [
-            ":s"
-        ]
-        # Pledge MFD's way of parsing time-related variables in a .cff
-        for time_label in "tref", "tp":
-            signal_parameter_formats[
-                signal_parameter_labels.index(time_label)
-            ] = ":10.6f"
```

Comment:
Do you remember why `.6f` is used here, while lal internally stores nanoseconds (which would be `.9f`)?

Reply:
> Should we take the opportunity to deliver on the `This will allow us to get rid of the get_single_config_* family` suggestion? Anything missing to use your simpler function in the standard Writer class instead of those?

I'll work on that in #146 

> Do you remember why `.6f` is used here, while lal internally stores nanoseconds (which would be `.9f`)?
IIRC it was written like that since the beginning... anyways, I just changed it in 93a7517 and it doesn't seem to produce any harm,

Reply:
Ok great!